cmd/dptp-controller-manager/main.go

```diff
@@ -285,7 +284,6 @@ func main() {
 
 		options := controllerruntime.Options{
 			DryRunClient: opts.dryRun,
-			Logger:       ctrlruntimelog.NullLogger{},
```

Comment:
According to 
https://pkg.go.dev/sigs.k8s.io/controller-runtime@v0.12.2/pkg/manager#Options

and

https://pkg.go.dev/sigs.k8s.io/controller-runtime@v0.12.2/pkg/log#pkg-variables

The default logger should be logging nothing.